README.md

```diff
@@ -10,6 +10,8 @@ Available on [pypi](https://pypi.python.org/pypi/vega) and [Conda Forge](https:/
 
 ## Install and run
 
+If you are using a python kernel different than the one used by the notebook server, you will need to install `vega` on both environments.
+
```

Comment:
This is not entirely correct... for rendering via altair renderer entrypoints, vega only needs to be installed in the frontend. It's only if you want to directly import the python library in your kernel that you need to also install it in your kernel.

Reply:
I see. I am still new to altair/vega, it will take some time to fully understand the inner workings. Feel free to close this PR or update changes as necessary.

Reply:
Thanks for sending the initial pull request. Feel free to send a follow up later.